.gitignore

```diff
@@ -24,3 +24,5 @@ vendor/
 
 # Temporary files
 FRAMEWORKS.yaml
+
```

Comment:
not sure it has to be here, but in `node` directory

Reply:
never liked multiple `.gitignore`, but upto you 👍